ostd/src/mm/dma/mod.rs

```diff
@@ -1,72 +1,17 @@
 // SPDX-License-Identifier: MPL-2.0
 
-mod dma_coherent;
-mod dma_stream;
+//! Direct Memory Access (DMA) memory management.
```

Comment:
From OSTD users' perspective, all DMA-related APIs fall under this module and it is a sub-module of the memory management module. So we can safely remove "memory management" from the summary sentence.